### PR TITLE
feat: rtl toolbar item setting

### PR DESCRIPTION
**Pull request by zoli**
Enable or disable RTL toolbar items through settings. Currently, they are only enabled when the default text direction setting is set to RTL. This separates the option to an independent setting.

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/direction_setting.dart [hunk 1]
```diff
@@ -141,3 +141,29 @@ class TextDirectionSetting extends StatelessWidget {
...
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ThemeSettingEntryTemplateWidget(
+      label: LocaleKeys.settings_appearance_enableRTLToolbarItems.tr(),
+      trailing: [
+        Switch(
+          key: const ValueKey('enable_rtl_toolbar_items_switch'),
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/direction_setting.dart:156]: define the string as const value

Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/direction_setting.dart:156]: and mark it as onlyvisiblefortesting

Comment (zoli) [reply frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/direction_setting.dart:156]: Done the visible for testing part. But the string was const before I didn't get which string you meant.

**Review by skydiver1980 — APPROVED** [review]

**Comment by LucasXu0** [comment]
@zoli, I resolved the conflicts.

**Comment by zoli** [comment]
> @zoli, I resolved the conflicts.

Is there anything I should do to get this merged?

**Pull request merged**